src/Custom-sample.h

```diff
@@ -77,16 +77,16 @@
 #define DEFAULT_CONTROLLER_PASS    ""                                       // Default controller Password
 
 // using a default template, you also need to set a DEFAULT PROTOCOL to a suitable MQTT protocol !
-#define DEFAULT_PUB         "sensors/espeasy/%sysname%/%tskname%/%valname%" // Enter your pub
-#define DEFAULT_SUB         "sensors/espeasy/%sysname%/#"                   // Enter your sub
-#define DEFAULT_SERVER      "192.168.0.8"                                   // Enter your Server IP address
+#define DEFAULT_PUB         "/%sysname%/%tskname%/%valname%"                // Enter your pub
+#define DEFAULT_SUB         "/%sysname%/#"                                  // Enter your sub
```

Comment:
Having a leading `/` in a MQTT topic is not to be adviced as it is adding an extra level to the topics.
On top of that, do not change it in Custom-sample.h